### PR TITLE
Load wkt

### PR DESCRIPTION
**Pull request by maltaesousa**
Loads WKT from textarea

### CODE REVIEW

File: js/sitnlayers.js [hunk 1]
```diff
@@ -1,27 +1,28 @@
+/* global proj4, ol, $ */
 (function (window) {
-  function sitnLayers() {
-    var sitnLayers = {};
-    const _extent = [2420000, 1030000, 2900000, 1350000];
-    const _crs = 'EPSG:2056';
-    const _WMTSurl= 'https://sitn.ne.ch/web_getcapabilities/WMTSGetCapabilities95.xml'
+  function sitnLayers () {
+    var sitnLayers = {}
```
Comment (kalbermattenm) [inline js/sitnlayers.js:4]: Uh ? Are semi-colons not needed anymore ?

File: js/sitnlayers.js [hunk 2]
```diff
@@ -32,26 +33,40 @@
...
 
     sitnLayers.view = new ol.View({
       projection: _projection,
       center: ol.proj.fromLonLat([6.80, 47.05], _projection),
       zoom: 4
-    });
+    })
 
     sitnLayers.map = new ol.Map({
       target: 'sitn-map',
```
Comment (kalbermattenm) [inline js/sitnlayers.js]: Shouldn't the target be configurable ?

I would rather call `sitnLayers('target-map')` to show them how to load it dynamically into a given `<div>`...

File: js/sitnlayers.js (hunk 1 above)
Comment (maltaesousa) [reply js/sitnlayers.js:4]: I'm always forgetting them so I decided to use StandardJS: https://standardjs.com/ linting.
If you want me to use another one, tell me which one and it'll be a pleasure :)

File: js/sitnlayers.js (hunk 2 above)
Comment (maltaesousa) [reply js/sitnlayers.js]: ok, thanks!

**Pull request merged**